### ISSUE TITLE
Implement the manifest model and serializers

### ISSUE THREAD

**Issue by dpgraham4401** [dpgraham4401/haztrak#7]
Haztrak needs to be able to consume and store manifest data from rcrainfo. At a high level this includes

- [x] pulling from rcrainfo (or third party)
- [ ] deserializing to native python objects
- [ ] how this data is represented in persistent storage

In addition, haztrak should be able to serve manifests as well
- [x] serializing to json
- [ ] exposing an (RESTful) API

pulling is easy, that's already implemented by the emanfiest client library which pulls the manifests down as json.

Model and Serializer implementation:
- [x] created date
- [x] update Date
- [x] mtn
- [x] submission type
- [x] signature status
- [x] origin type
- [x] shipped date
- [x] potential ship date
- [x] received date
- [x] certified date
- [x] certified by
- [x] generator
- [x] **transporters**
- [x] tsd
- [ ] ~~broker~~
- [x] **waste**
- [x] rejection boolean
- [x] rejection info
- [x] import boolean
- [x] import info
- [x] contains rejection or residue
- [ ] ~~printed doc~~
- [ ] ~~form doc~~
- [ ] ~~additional info~~
- [x] correction info
- [x] PPC status
- [x] locked
- [x] locked reason

A lot of the fields lower on this list are lower priority and we will continue to simply take the json from RCRAInfo and store it as text. 

Transporter and waste will be tricky ones and are the last thing to really implement before we can move on to other things. 

In the future, the ability to store PDFs will be required but for now we're not focused on that.

**Comment by dpgraham4401** [comment]
simple is nice, but functional is more of a priority. It's looking like the second option will allow better control of things and allow us to lean on django's generic views. option 1, while simple, is likely not scalable.
